code/integration-tests/runtime-tests/test/tests/assets/rpcAssetsTests.ts

```diff
@@ -49,30 +49,29 @@ describe("[SHORT] rpc.assets Tests", function () {
   it("rpc.assets.listAssets Tests", async function () {
```

Comment:
@dzmitry-lahoda these are the changes we talked about earlier today to make this test more robust for first release.